### PR TITLE
verify shellcheck

### PR DESCRIPTION
**Pull request by pohly**
This will be used for the Prow test script and could also be applied to other scripts, like for example the deploy.sh script in csi-driver-host-path.

### CODE REVIEW

File: util.sh [hunk 1]
```diff
@@ -0,0 +1,839 @@
...
+  echo -n "${bin}"
+}
+
+kube::util::find-binary() {
+  kube::util::find-binary-for-platform "$1" "$(kube::util::host_platform)"
+}
+
+# Run all known doc generators (today gendocs and genman for kubectl)
+# $1 is the directory to put those generated documents
+kube::util::gen-docs() {
```
Comment (msau42) [inline util.sh]: Many of these functions do not seem relevant for us. Can we remove the unused ones?

File: verify-shellcheck.sh [hunk 2]
```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env bash
```
Comment (msau42) [inline verify-shellcheck.sh]: Can you add a README describing what are we validating with this script?

File: util.sh (hunk 1 above)
Comment (pohly) [reply util.sh]: Okay.

File: verify-shellcheck.sh (hunk 2 above)
Comment (pohly) [reply verify-shellcheck.sh]: Okay.

File: util.sh (hunk 1 above)
Comment (pohly) [reply util.sh]: I kept some smaller functions which may become useful.

**Comment by pohly** [comment]
@msau42 updated.

**Pull request merged**